api/spec/controllers/mno_enterprise/jpi/v1/subscriptions_controller_spec.rb

```diff
@@ -1,12 +1,17 @@
 require 'rails_helper'
 
 module MnoEnterprise
-  describe Jpi::V1::SubscriptionsController, type: :controller do
+  describe Jpi::V1::SubscriptionsController, type: :controller, ignore: true do
```

Comment:
It looks like a Json ApiClient bug.

```    def initialize(params = {})
      @persisted = nil
      self.links = self.class.linker.new(params.delete("links") || {})
      self.relationships = self.class.relationship_linker.new(self.class, params.delete("relationships") || {})
      self.class.associations.each do |association|
        if params.has_key?(association.attr_name.to_s)
          set_attribute(association.attr_name, association.parse(params[association.attr_name.to_s]))
        end
      end
      self.attributes = params.merge(self.class.default_attributes)
      self.class.schema.each_property do |property|
        attributes[property.name] = property.default unless attributes.has_key?(property.name) || property.default.nil?
      end
    end```

there is no trace of parse method on the Assocation Class
```
module JsonApiClient
  module Associations
    class BaseAssociation
      attr_reader :attr_name, :klass, :options
      def initialize(attr_name, klass, options = {})
        @attr_name = attr_name
        @klass = klass
        @options = options
      end

      def association_class
        @association_class ||= Utils.compute_type(klass, options.fetch(:class_name) do
          attr_name.to_s.classify
        end)
      end

      def data(url)
        from_result_set(association_class.requestor.linked(url))
      end

      def from_result_set(result_set)
        result_set.to_a
      end
    end
  end
end
```

```module JsonApiClient
  module Associations
    module HasMany
      extend ActiveSupport::Concern

      module ClassMethods
        def has_many(attr_name, options = {})
          self.associations = self.associations + [HasMany::Association.new(attr_name, self, options)]
        end
      end

      class Association < BaseAssociation
      end
    end
  end
end```